RELEASE.md

```diff
@@ -43,7 +43,11 @@ makes sure that we don't have any extra tags or commits in our repo (especially
 since we will push our tags later in the process), and that we are on the correct branch. The script creates a conda env, pulls down a git checkout with the
 appropriate branch, and installs JupyterLab with `pip install -e .`.
 
-`source scripts/release_prep.sh <branch_name>`
+If your sh compatible shell is not set up with conda yet, do that first:
+
+```bash
+source scripts/release_prep.sh <branch_name>
```

Comment:
We explicitly use this regardless to have a completely clean environment.

Reply:
sorry my note meant to first setup conda with your shell, before running this script. ill update it.